Criado putSensores.js

FranciscoOssian opened the pull request:
Importante que pode ser usado mesmo para cadastrar 1 sensor.

FranciscoOssian committed:
Criado putSensores

Importante que pode ser usado mesmo para cadastrar 1 sensor.

rafaelalmeida2909 commented:
Só uma correção put sensores na verdade é post e ficou acordado de usarmos camelcase ao invés de underscore.

FranciscoOssian committed:
update PutSensore.js (PostSensores.js)

ainda falta sobre calmel case

FranciscoOssian committed:
Mudado para CamelCase

Embora onde foi usado underscore não reflita sobre a API mais sim o que ela recebe, entendo que é importante para o exemplo ser semelhante com o proposto por nós.

rafaelalmeida2909 committed:
postSensores.js

Criação da Rota 6

rafaelalmeida2909 committed:
Update postSensores.js
